dev/generate-fuzz-hypothesis-tests.py

```diff
@@ -0,0 +1,121 @@
+import os
+from hypothesis import strategies as st
+from hypothesis.strategies import composite
+import yaml
+
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+
+<<<<<<< HEAD
```

Comment:
Are these git conflict markers supposed to be here?

Reply:
sorry overlooked that, just pushed another commit